code/src/App.js

```diff
@@ -1,9 +1,138 @@
-import React from 'react'
+import React, { useState } from "react";
+import "./index.css";
+import { Summary } from "Summary";
 
 export const App = () => {
+  const sizeComp = ["0-10 people", "10-50 people", "50+ people"];
```

Comment:
If all have "people" after you can add it to the output
```suggestion
  const ages = ["0-10", "10-50", "50+"];
```